Libplanet/Net/SwarmOptions.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using Libplanet.Blocks;
+using Libplanet.Tx;
+
+namespace Libplanet.Net
+{
+    public class SwarmOptions
```

Comment:
How about using `struct` instead of `class`?

Reply:
I made it to a `class` to use initializers. Would it be better to change it to `struct`?

Reply:
Maybe... because it seems just data class. but on the other way, it isn't a big deal.